File: C/7kyu/Sum of the First Nth Term of Series/solution.c
#include <stddef.h>
#include <stdlib.h>

char *series_sum(const size_t n) {
  double result = 0;
  int base = 1;
  char *ret = (char *) malloc(sizeof(char) * 50);
  
  for(int i = 0; i < n; i++) {
    result = result + (1.0/base);
    base += 3;
  }
  
  snprintf(ret, 50, "%.2f", result);
  return ret;
}

File: PHP/7kyu/Sum of the First Nth Term of Series/solution.php
<?php
function series_sum($n) {
  $base = 1;
  $sum = 0;
  
  while($n) {
    $sum = $sum + 1.0/$base;
    $base += 3;
    --$n;
  }
  return number_format($sum, 2);
}
?>